test/Tests/src/Semantic/Any_Spec.enso

```diff
@@ -42,19 +42,64 @@ spec =
             My_Type.Value 33 . x . should_equal "Any:(My_Type.Value 33)"
             With_X.Value 44 . x . should_equal "With_X:(With_X.Value 44)"
             With_Y.Value 44 . x . should_equal "With_Y:With_Y(44)"
-            Any.x . to_text . should_equal "Any.type.x[Any_Types.enso:6-26]"
             My_Type.x . should_equal "Any:My_Type"
             With_X.x . to_text . should_equal "Any:With_X"
-            With_X.y . to_text . should_equal "With_X.type.y[Any_Types.enso:12-32]"
+            Meta.is_a With_X.y Function . should_be_true
             With_Y.x . to_text . should_equal "Any:With_Y"
-            With_Y.y . to_text . should_equal "With_Y.type.y[Any_Types.enso:18-38]"
+            Meta.is_a With_Y.y Function . should_be_true
             With_X.to_text . to_text . should_equal "With_X"
             With_Y.to_text . to_text . should_equal "With_Y"
+            Any.x . to_text . should_equal "Any:Any"
+            Any.x self=Any . to_text . should_equal "Any:Any"
             Any.x self=With_X . should_equal "Any:With_X"
             Any.x self=With_Y . should_equal "Any:With_Y"
             Any.x (My_Type.Value 22) . should_equal "Any:(My_Type.Value 22)"
             Any.x (With_X.Value 22) . should_equal "Any:(With_X.Value 22)"
             Any.x (With_Y.Value 22) . should_equal "Any:With_Y(22)"
             Date.to_display_text . should_equal "Date"
 
+        Test.specify "static method calls on Any should have defaulted self argument to Any" <|
```

Comment:
Regardless of my other comments, these tests are the most important thing. If they work & define the expected behavior, then we have a system we wanted.